### PR TITLE
ADD Project support

### PR DESCRIPTION
**Pull request by jonathanlurie**

### CODE REVIEW

File: nexussdk/project.py [hunk 1]
```diff
@@ -2,26 +2,165 @@
...
+    if not is_response_valid(response_raw):
+        raise Exception("Invalid http request for " + response_raw.url +
+                        " (Status " + str(response_raw.status_code) + ")" + "\n" +
+                        response_raw.text)
+
+    response_obj = json.loads(response_raw.text)
+    return response_obj
+
+
+def create(org_label, project_label, data=None):
```
Comment (MFSY) [inline nexussdk/project.py]: I would call this argument config instead of data.

File: nexussdk/project.py [hunk 2]
```diff
@@ -2,26 +2,165 @@
...
+
+    :param org_label: The label of the organization to create the project in
+    :param project_label: The label of the project to add
+    :param data: OPTIONAL if provided, must be a dictionary. Can include data such as `name`, `base` or `prefixMapping`
+    see https://bluebrain.github.io/nexus/docs/api/admin/admin-projects-api.html#create-a-project for more info
+    :return: The payload from the Nexus API as a dictionary. This contains the Nexus metadata of the project
+    """
+    org_label = urllib.parse.quote_plus(org_label)
+    project_label = urllib.parse.quote_plus(project_label)
+    url = "/projects/" + org_label + "/" + project_label
```
Comment (MFSY) [inline nexussdk/project.py]: I would not call this 'url' but 'path' instead as the actual url is built through the http helper.

File: nexussdk/project.py (hunk 1 above)
Comment (jonathanlurie) [reply nexussdk/project.py]: true, it's more a config than actual data. Changing the code.

File: nexussdk/project.py (hunk 2 above)
Comment (jonathanlurie) [reply nexussdk/project.py]: true. but since i use that everywhere, i'll fix that in the next PR

Comment (jonathanlurie) [reply nexussdk/project.py]: (just to not have a PR that targets a lot of different stuff)

**Pull request closed without merging**